docs/resources/virtual_server.md

```diff
@@ -53,81 +78,128 @@ resource "anxcloud_virtual_server" "example" {
-- `custom_name` - Virtual server custom name.
-- `location_code` - Location code.
-- `location_country` - Location country.
-- `location_name` - Location name.
-- `cpu` - Number of CPUs.
-- `cores` - Number of CPU cores.
-- `ram` - Memory in MB.
-- `disks_number` - Number of the attached disks.
-- `disks_info` - Disks info. See [disks info](#disks-info) below for details. 
-- `network` - Network interfaces. See [network interfaces](#network-interfaces) below for details.
-- `guest_os` - Guest operating system.
-- `version_tools` - Version tools.
-- `guest_tools_status` - Guest tools status.
-
-### Disks info
-
-- `disk_id` - Disk identifier.
-- `disk_gb` - Size of the disk in GB.
-- `disk_type` - Disk type.
-- `iops` - Disk input/output operations per second.
-- `latency` - Disk latency.
-- `storage_type` - Disk storage type.
-- `bus_type` - Disk device bus type.
-- `bus_type_label` - Disk device bus type label.
-
-### Network interfaces
-
-- `id` - Network interface card identifier.
-- `ip_v4` - List of IPv4 addresses attached to the interface.
-- `ip_v6` - List of IPv6 addresses attached to the interface.
-- `nic` - NIC type number.
-- `vlan` - VLAN identifier..
-- `mac_address` - MAC address of the NIC.
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `cpus` (Number) Amount of CPUs.
+- `disk` (Block List, Min: 1) Virtual Server Disks (see [below for nested schema](#nestedblock--disk))
+- `hostname` (String) Virtual server hostname.
+- `location_id` (String) Location identifier.
+- `memory` (Number) Memory in MB.
+- `template_id` (String) Template identifier.
+- `template_type` (String) OS template type.
+
+### Optional
+
+- `boot_delay` (Number) Boot delay in seconds. Example: (0, 1, …).
+- `cpu_performance_type` (String) CPU type. Example: (`best-effort`, `standard`, `enterprise`, `performance`), defaults to `standard`.
+- `critical_operation_confirmed` (Boolean) Confirms a critical operation (if needed). Potentially dangerous operations (e.g. resulting in data loss) require an additional confirmation. The parameter is used for VM UPDATE requests.
+- `dns` (List of String) DNS configuration. Maximum items 4. Defaults to template settings.
+- `enter_bios_setup` (Boolean) Start the VM into BIOS setup on next boot.
+- `force_restart_if_needed` (Boolean) Certain operations may only be performed in powered off stat.Such as: shrinking memory, shrinking/adding cpu, removing disk, scale a disk beyond 2 GB. Passing this value as true will always execute a power offand reboot request after completing all other operations. Without this flag set to true scaling operations requiring a reboot will fail.
+- `id` (String) The ID of this resource.
+- `network` (Block List) Network interface (see [below for nested schema](#nestedblock--network))
+- `password` (String, Sensitive) Plaintext password. Example: ('!anx123mySuperStrongPassword123anx!', 'go3ju0la1ro3', …). USE IT AT YOUR OWN RISK! (or SSH key instead).
```

Comment:
Lets change the "USE IT AT..." to "For systems that support it, we strongly recommend using a SSH key instead."